modal/functions.py

```diff
@@ -181,7 +185,7 @@ async def poll_function(self, timeout: Optional[float] = None):
         )
 
         if len(items) == 0:
-            raise TimeoutError()
+            raise TimeoutError("Function call has not finished")
```

Comment:
Is it right that we're reporting a timeout _on the `get`_ operation, rather than on the underlying function call? We may want to reword this to make that a little more clear.

Reply:
yeah, should be reporting on the get operation